ui/packages/atlasmap/src/UI/MappingTransformation.tsx

```diff
@@ -144,6 +146,16 @@ export const MappingTransformation: FunctionComponent<IMappingTransformationProp
               />
             )}
           </FormGroup>
+        ) : (
```

Comment:
Can you paste a screen shot as to what this looks like.  Also I'm not sure I like `Delimit empty values`.  It's terse which is typically a good thing but maybe too much.  Maybe:
`Add the specified delimiter for empty values.`

Reply:
I don't want to take up too much space for this, as there is already very little real estate to work with or that we are forcing user to expand to.